Exporting-Server/server/spring-boot-jersey/src/main/java/com/server/reveal/RevealDashboardProvider.java

```diff
@@ -12,8 +12,7 @@ public class RevealDashboardProvider implements IRVDashboardProvider {
 
     @Override
     public InputStream getDashboard(IRVUserContext userContext, String dashboardId) throws IOException {
-        InputStream dashboardStream = new FileInputStream("dashboards/" + dashboardId + ".rdash");
```

Comment:
Why did you move the dashboard and change the loading strategy?

Reply:
I don't use Eclipse. I use VS Code and it works there. Loading dashboards from a resource is not the same as loading from a folder on the server. I'm closing this since it doesn't really fix anything.